src/main/java/io/zold/api/WalletsIn.java

```diff
@@ -35,11 +36,17 @@
 import org.cactoos.scalar.IoCheckedScalar;
 import org.cactoos.scalar.StickyScalar;
 import org.cactoos.scalar.SyncScalar;
+import org.cactoos.text.JoinedText;
 
 /**
  * Wallets in path.
  *
  * @since 0.1
+ * @todo #12:30min Make uniform the way file extensions are handled
```

Comment:
@victornoel not sure why we need a separate task here, should be fairly straightforward. Refactor the class by storing the extension as an instance variable, move the filter from the instance to a local variable to `iterator()`, and use the extension both in `iterator` and in `create`.

Reply:
@victornoel why not in the scope? It's at least somewhat related, it affects your implementation and the way wallets are fetched. Aside from that, it's an internal detail that's a) not defined in some architectural documentation (as far as I know), and b) easy to fix. An extra task seems superfluous to me.

Reply:
@carlosmiranda because I spent the allotted time on this, and the original issue wasn't about having a coherent naming of the wallet files, it was about creating a new wallet.

Reply:
@victornoel ok, as far as changes to this go, I'm all good. I'll approve it and let @llorllale decide what to do further.

Reply:
@victornoel I agree with @carlosmiranda on this: it was clear that the file extension for wallets is `.z` and it also looks like something easily handled internally as said [here](https://github.com/zold-io/java-api/pull/55/files#r207752568).

Reply:
@carlosmiranda @llorllale ok I will implement that